app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :likeable, polymorphic: true
```

Comment:
can a "Like" be created without the "likable" content or the "user"? we should probably have validations on these foreign_key fields 👍